### PR TITLE
Minor changes related to localisation...

### PR DESCRIPTION
**Pull request by Pizzacus**
- Replaced the function `addCommas()` by the function `toLocaleString()`,  which format numbers based on the users location (example: in England, 5678 becomes 5,678, while in France, 5678 becomes 5 678, because spaces are used to format numbers in France)
- Removed the special behaviour for 301+ views, because this bug doesn't exist anymore (https://twitter.com/YTCreators/status/628958720953819136/)
- The search bar placeholder in the extension popup is now translated
- The "Add to YouTube Notification" button is now translated
- Improved the french translation a little bit

Also, It seems like my editor (atom) decided to remove indentation on empty lines... I didn't do that on purpose, I didn't know it could do it

So if you're not okay with that (which I totally understand, I wouldn't like it if someone messed up with my code writing style either...), I can try to revert the empty lines and see how I can prevent this problem from happening again...

### CODE REVIEW

**Comment by Pizzacus** [comment]
Wait... why is travis trying to build this pull request and not the previous one ? And why did it fail, i didn't do anything that could make it fail, i tested it, it works for me...

**Comment by Wassup789** [comment]
Don't worry about the empty line indentation that much, that was when I was using n++ for editing, now I use IntellijIdea (although looking through the changes were a bit of a pain, lol). Also, ignore the Travis checks, I never actually set that up yet. I never actually knew about toLocalesString() until today, might have to update my own apps because of how useful that is.

I would actually rename the locale prefixes from "yt_" to "contentScript_" for some consistencies in the locales and also add a "getString" function similar to the ones in background.js and settings_edit.js for future uses if needed.

File: _locales/en/messages.json [hunk 1]
```diff
@@ -370,5 +370,17 @@
...
+		"description": "The add to YouTube Notifications button on YouTube",
+		"message": "Add to YouTube Notifications"
+	},
+	"yt_channelAdded": {
+		"description": "The add to YouTube Notifications button on YouTube when clicked",
+		"message": "Added to YouTube Notifications"
+	},
+	"yt_removeChannel": {
+		"description": "The remove from YouTube Notifications button on YouTube displayed after the channel is added",
+		"message": "Remove from YouTube Notifications"
```
Comment (Wassup789) [inline _locales/en/messages.json:384]: Change prefix to contentScript_

**Comment by Pizzacus** [comment]
done

**Pull request merged**